README.md

```diff
@@ -100,6 +100,7 @@ A curated list of awesome [Apache Spark](https://spark.apache.org/) packages and
 ### Utilities
 
 * [silex](https://github.com/willb/silex) - A bunch of tools varying from ML extensions to additional RDD methods.
+* [sparkly](https://github.com/Tubular/sparkly) - Helpers & syntax sugar for PySpark
```

Comment:
- Shouldn't it be _syntactic sugar_ instead of  _syntax sugar_?
- Missing full stop.